src/methods/clipping/difference.jl

```diff
@@ -0,0 +1,292 @@
+# #  Difference Polygon Clipping
+export difference
+
+# The 'difference' function returns the difference of two polygons. Note that this file
```

Comment:
For difference, union, and intersection, let's mention the algorithm name and add in the link to the paper